ignite/cmd/network.go

```diff
@@ -235,3 +231,41 @@ func getNetworkCosmosClient(cmd *cobra.Command) (cosmosclient.Client, error) {
 
 	return *cosmos, nil
 }
+
+func flagSetSpnNodeAddress() *flag.FlagSet {
+	fs := flag.NewFlagSet("", flag.ContinueOnError)
+	fs.String(flagSPNNodeAddress, spnNodeAddressNightly, "SPN node address")
+	return fs
+}
+
+func flagSetSpnFaucetAddress() *flag.FlagSet {
+	fs := flag.NewFlagSet("", flag.ContinueOnError)
+	fs.String(flagSPNFaucetAddress, spnFaucetAddressNightly, "SPN faucet address")
+	return fs
+}
```

Comment:
While you're at it, you can merge those 2 functions and returns a single flagset that contains the 2 flags.

Reply:
Modified per your suggestion. Thanks!